From 01201f6c94752758c2d4ffe6c69a0d42d132643b Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Wed, 18 Dec 2024 14:02:35 +0100
Subject: [PATCH] arm64: dts: st: set SDMMC max frequency to 166 MHz on
 stm32mp23

Set SDMMC max frequency to 166 MHz on stm32mp23.

Change-Id: Ief1b69f39f9169c2358cc3718f520f50b60d53dc
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/426126
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
---
 arch/arm64/boot/dts/st/stm32mp231.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp231.dtsi b/arch/arm64/boot/dts/st/stm32mp231.dtsi
index 4209219a5d13..1b4a2431be01 100644
--- a/arch/arm64/boot/dts/st/stm32mp231.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp231.dtsi
@@ -1974,7 +1974,7 @@ sdmmc1: mmc@48220000 {
 				resets = <&rcc SDMMC1_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				access-controllers = <&rifsc 76>;
 				power-domains = <&CLUSTER_PD>;
 				status = "disabled";
@@ -1990,7 +1990,7 @@ sdmmc2: mmc@48230000 {
 				resets = <&rcc SDMMC2_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				access-controllers = <&rifsc 77>;
 				power-domains = <&CLUSTER_PD>;
 				status = "disabled";
@@ -2006,7 +2006,7 @@ sdmmc3: mmc@48240000 {
 				resets = <&rcc SDMMC3_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				access-controllers = <&rifsc 78>;
 				power-domains = <&CLUSTER_PD>;
 				status = "disabled";
-- 
2.50.1

